op-batcher/flags/flags.go

```diff
@@ -186,6 +186,12 @@ var (
 		Value:   false,
 		EnvVars: prefixEnvVars("PREFER_LOCAL_SAFE_L2"),
 	}
+	// HA with rollup boost enabled flags
+	DAUpdateEndpointsFlag = &cli.StringSliceFlag{
+		Name:    "da-update-endpoints",
+		Usage:   "Comma-separated list of endpoints to distribute DA configuration updates to",
+		EnvVars: prefixEnvVars("DA_UPDATE_ENDPOINTS"),
+	}
```

Comment:
I think what's described as "DA size configuration distribution" here is what we just call "throttling". If that's correct, I would prefer to rename everything to keep with the existing semantics, like
```suggestion
	// HA with rollup boost enabled flags
	ThrottlingEndpointsFlag = &cli.StringSliceFlag{
		Name:    "throttling-endpoints",
		Usage:   "Comma-separated list of endpoints to throttle in addition to the active sequencer",
		EnvVars: prefixEnvVars("THROTTLING_ENDPOINTS"),
	}
```
This would also translate to all other variable and function names etc.